test/unit/whenKeyTest.js

```diff
@@ -0,0 +1,31 @@
+const test = require('tape');
+const helpers = require('../helpers/createDanceAPI');
+
+test('make sure we are checking all relevant keys', async t => {
+  const nativeAPI = await helpers.createDanceAPI();
+  const p5 = nativeAPI.p5_;
+
+  // simulate key presses
+  p5._onkeydown({which: p5.KEY.LEFT_ARROW});
+  p5._onkeydown({which: p5.KEY.SPACE});
+  p5._onkeydown({which: p5.KEY.ENTER});
+  p5._onkeydown({which: p5.KEY.A});
+  p5._onkeydown({which: p5.KEY.Z});
+  p5._onkeydown({which: p5.KEY['0']});
+  p5._onkeydown({which: p5.KEY['9']});
+  p5.readPresses();
```

Comment:
You could make the argument that this test is overly brittle, as we're using p5 internals to simulate our presses, but I think this is likely still better than nothing.

Reply:
I wonder if it's better to call `nativeAPI.onKeyDown` here?

Reply:
Agree entirely :)